src/CSSfile/main.css

```diff
@@ -0,0 +1,831 @@
+.hvr-back-pulse-href:link, .hvr-back-pulse-href:visited {
```

Comment:
i am using scss. 
u mean gitignore the css file
because this css file is generated via gulp only and the other one after after css-minification.